File: implementation/scipamato/scipamato-core-entity/src/test/java/ch/difty/scipamato/core/entity/keyword/KeywordTranslationTest.java
package ch.difty.scipamato.core.entity.keyword;

import static org.assertj.core.api.Assertions.assertThat;

import org.junit.jupiter.api.Test;

class KeywordTranslationTest {

    @Test
    void displayValue() {
        KeywordTranslation ntt = new KeywordTranslation(1, "de", "topic1", 1);
        assertThat(ntt.getDisplayValue()).isEqualTo("de: topic1");
    }
}

File: implementation/scipamato/core/entity/src/test/kotlin/ch/difty/scipamato/core/entity/keyword/KeywordTranslationTest.kt
package ch.difty.scipamato.core.entity.keyword

import org.assertj.core.api.Assertions.assertThat

import org.junit.jupiter.api.Test

internal class KeywordTranslationTest {

    @Test
    fun displayValue() {
        val ntt = KeywordTranslation(1, "de", "topic1", 1)
        assertThat(ntt.displayValue).isEqualTo("de: topic1")
    }
}